test/samples/SceneWriter.data.ts

```diff
@@ -1,77 +1,76 @@
-export const boxSample = `
-const box = new Entity()
+export const boxSample = `const box = new Entity()
+engine.addEntity(box)
 const boxShape = new BoxShape()
 box.addComponentOrReplace(boxShape)
 const transform = new Transform({
   position: new Vector3(5, 0, 5),
   rotation: new Quaternion(0, 0, 1, 0),
   scale: new Vector3(1, 1, 1)
 })
-box.addComponentOrReplace(transform)
-engine.addEntity(box)`
+box.addComponentOrReplace(transform)`
 
-export const gltfSample = `
-const skeleton = new Entity()
+export const gltfSample = `const skeleton = new Entity()
+engine.addEntity(skeleton)
 const gltfShape = new GLTFShape('./Skeleton.gltf')
 skeleton.addComponentOrReplace(gltfShape)
 const transform = new Transform({
   position: new Vector3(0, 0, 0),
   rotation: new Quaternion(0, 2, 1, 0),
   scale: new Vector3(1, 1, 1)
 })
-skeleton.addComponentOrReplace(transform)
-engine.addEntity(skeleton)`
+skeleton.addComponentOrReplace(transform)`
 
 export const ntfShape = `
 const kitty = new Entity()
+engine.addEntity(kitty)
 const nftShape = new NFTShape('ethereum://0x06012c8cf97BEaD5deAe237070F9587f8E7A266d/38376')
-kitty.addComponentOrReplace(nftShape)
-engine.addEntity(kitty)`
+kitty.addComponentOrReplace(nftShape)`
 
 export const sphereAndBoxSample = `
 const sphere = new Entity()
+engine.addEntity(sphere)
 const sphereShape = new SphereShape()
 sphere.addComponentOrReplace(sphereShape)
-engine.addEntity(sphere)
 
 const box = new Entity()
+engine.addEntity(box)
 const boxShape = new BoxShape()
-box.addComponentOrReplace(boxShape)
-engine.addEntity(box)`
+box.addComponentOrReplace(boxShape)`
 
 export const parentSample = `
 const sphere = new Entity()
+engine.addEntity(sphere)
 const sphereShape = new SphereShape()
 sphere.addComponentOrReplace(sphereShape)
-engine.addEntity(sphere)
 
 const box = new Entity()
+engine.addEntity(box)
```

Comment:
This seems to be a regression (it was fixed by #9) an entity with a parent shouldn't be added to the engine. This doesn't throw but it prints a warning on the console. And since all the entities in a builder scene are children of the `sceneEntity`, this outputs thousands of warnings in the console for each parcel deployed through the builder.

Reply:
That warning is now silenced: https://github.com/decentraland/kernel/blob/6a4269763715440e0b5f64e2cbbaf6f59f8021cc/packages/shared/logger.ts#L20

Reply:
patched by https://github.com/decentraland/kernel/blob/6a4269763715440e0b5f64e2cbbaf6f59f8021cc/packages/shared/logger.ts#L20